tutorials/kernelshap_tabular_weather.ipynb

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
<p>Line 11 is mysterious. See comment in other notebook. Better remove it.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Solved.</p>